python/lsst/obs/hsc/makeTransmissionCurves.py

```diff
@@ -41,16 +41,14 @@ def getLongFilterName(short):
     """Return a long HSC filter name (e.g. 'HSC-R') that's usable as a data ID
     value from the short one (e.g. 'r') declared canonical in afw.image.Filter.
     """
-    HscMapper.addFilters()
     if short.startswith("HSC"):
         return short
     if short.startswith("NB") or short.startswith("IB"):
         num = int(short[2:].lstrip("0"))
         return "%s%04d" % (short[:2], num)
-    f = Filter(short)
-    for a in f.getAliases():
-        if a.startswith("HSC") or a.startswith("NB") or a.startswith("IB"):
-            return a
+    for filter in hscFilters.HSC_FILTER_DEFINITIONS:
+        if short == filter.afw_name or short == filter.band:
```

Comment:
I thought we were planning to deprecate `afw_name`. Is that not possible, or is this a temporary fix?

Reply:
My change here was to get rid of the call to `Filter` and instead just use the FilterDefinitions. I've filed DM-28334 about this issue: the correct solution is to relabel the transmission curves with the physical filter name.